packages/asc-ui/src/components/Radio/Radio.stories.tsx

```diff
@@ -172,3 +175,34 @@ export const RadioLabelPositions = () => (
     </RadioGroup>
   </>
 )
+
+export const RadioWithRef = () => {
+  const ref = createRef<HTMLInputElement>()
+
+  useEffect(() => {
+    const element = ref.current
+
+    function onChange() {
+      // eslint-disable-next-line no-console
+      console.log('Value changed to option 2!')
+    }
+
+    element.addEventListener('change', onChange)
+
+    return () => element.removeEventListener('change', onChange)
+  }, [ref])
+
+  return (
+    <RadioGroup name="with-ref">
```

Comment:
The usage of id `with-ref` on the group and individual inputs would suggest all elements have a ref, while only the second one has a ref. Can you make this more clear to the user?

Reply:
Good catch, changed.